AddSevenPS/Library.fs

```diff
@@ -1,4 +1,37 @@
 ﻿namespace AddSevenPS
 
-type Class1() = 
-    member this.X = "F#"
+open System.Management.Automation
+
+open System.Management.Automation.Runspaces
+
+module AddSeven =
+    let addSeven num = num + 7
+     
+    // For testing purpose
+    //  Testing list of integers and then returning the list with solutions  
+    let add7 numList =
```

Comment:
For this, I like the idea but the naming convention doesn't convey that this function `add7` can take a list.

As per the [`dotNET` naming guidelines](https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/component-design-guidelines#naming-conventions) `addSevenToList` makes more sense. 

Please refactor :) 


`numList` is good

Reply:
Done